### PR TITLE
PERL_FEATURES implementation --- CI only, do not merge

### PR DESCRIPTION
**Pull request by akhuettel**

### CODE REVIEW

File: app-editors/padre/padre-1.0.0-r3.ebuild [hunk 1]
```diff
@@ -13,10 +13,9 @@ HOMEPAGE="https://padre.perlide.org/"
 
 SLOT="0"
 KEYWORDS="~amd64 ~riscv ~x86"
-IUSE=""
 
 # Test Deps
-TDEPEND="
+TESDEPEND="
```
Comment (thesamesam) [inline app-editors/padre/padre-1.0.0-r3.ebuild:18]: Can't we do TESTDEPEND? TESDEPEND just looks like a typo (sorry to bikeshed)

**Review by thesamesam — COMMENTED** [review]
I'm going to test this over the next few days with https://github.com/thesamesam/sam-gentoo-scripts/blob/main/sync/repo-patches. It looks sensible though

**Comment by akhuettel** [comment]
I think this works now as intended, at least in my chroot. (Needs some squashing at the end...)

**Comment by ConiKost** [comment]
I guess, that this can be closed?

**Comment by thesamesam** [comment]
Yes, thanks!

**Pull request closed without merging**